subnets.tf

```diff
@@ -10,6 +10,7 @@ resource "aws_subnet" "public" {
     Name        = "Public ${var.name} ${element(var.az_list, count.index)}"
     Owner       = "${var.owner_tag}"
     Environment = "${var.env_tag}"
+    Zone_type   = "public"
```

Comment:
My only nit is do we want to use `ZoneType`? `Zone_type` seems to be combining a couple different casing options (underscores are allowed though).

Reply:
I'm fine with camelcase